### PR TITLE
Outstanding changes from my fork

### PR DESCRIPTION
**Pull request by ephiax20**
# Module: 2
## Lab/Demo: All

Changes proposed in this pull request:

- Mainly click/select and type/enter changes.
- Other minor text formatting or layout changes.

### CODE REVIEW

File: Instructions/Labs/M2 Manage manufacturing processes.md [hunk 1]
```diff
@@ -3368,27 +3374,23 @@ You will have to do the following:
 
 12. Select **Show arrivals from lines**.
 
-13. Select **Journals**.
```
Comment (mstall1ngs) [inline Instructions/Labs/M2 Manage manufacturing processes.md:3371]: Update Step 13 to: Expand the Line details FastTab.
image
14. Select the **Dimension** tab
15. Select **Edit**.
16. Select to follow the link in the **Batch number** field
17. Select **New**
18. In the **Batch number** field, enter or select a value.
19. Select **Save**.
20. In the **Expiration date** field, enter a date.
Continue with renumbering. This exercise should have 32 steps after you've updated this.

**Review by mstall1ngs — CHANGES_REQUESTED** [review]
Thanks for the testing, Simon! Please review the changes and make updates prior to merging.

File: Instructions/Labs/M2 Manage manufacturing processes.md [hunk 2]
```diff
@@ -198,23 +198,23 @@ You will have to do the following:
...
+ 
+5.  Select **Save**.
+ 
+6.  Select the **Route** tab on the Action Pane then in the **Maintain** group, select **Route details**.
+ 
+7.  Select **New**.
+ 
+8.  In the **Operation** field, enter or select **Assembly**.
+ 
+9.  In the **Next** field, enter **20**.
```
[the remark this answers is not in the document]
Comment (ephiax20) [reply Instructions/Labs/M2 Manage manufacturing processes.md:217]: Hi,

The additional space is meant to align the Content for Tasks with more than 9 steps, to make it easier to read in the Markdown format. It doesn't actually affect the instructions when they're rendered in ALH instructions.

Example:

8. Test  <- single space here means the instruction is not aligned with the rest.
9.  Test <- double space here, I have added for Steps 1-9 so it aligns. 
10. Test <- single space here for Steps 10 onwards to keep everything aligned.

This is why I have done this. This approach is inconsistently used across all the Biz Apps courses I have tested so far. Some do, some don't. I am trying to standardize where possible. I feel it makes it easier to read and scan through the instructions when they're aligned.

File: Instructions/Labs/M2 Manage manufacturing processes.md [hunk 3]
```diff
@@ -1453,7 +1452,7 @@ You will have to do the following:
 
 3.  In the **Item number** field, enter or select **M12000**. 
 
-    Note: When you select **New**, you may need to change the dropdown in the selection menu to **Other**.
+    > Note: When you select **New**, you may need to change the dropdown in the selection menu to **Other**.
```
[the remark this answers is not in the document]
Comment (ephiax20) [reply Instructions/Labs/M2 Manage manufacturing processes.md:1455]: This use of greater-than > symbol here is not about describing UI. This is part of Markdown syntax to highlight the Note correctly, it changes the formatting and makes it easier to read the note.

This is about standardizing the markdown code. If I recall correctly, some notes have the > and some don't. I am trying to standardize whenever a Note appears in the instructions so they all look the same.

**Pull request closed without merging**